tracex_project/db_results/views.py

```diff
@@ -197,3 +199,119 @@ def create_bar_chart(data, x_title, y_title):
                 "showTips": False,
             },
         )
+
+
+class EvaluationView(FormView):
+    """View for displaying all extracted traces and DFG image with filter selection."""
+
+    form_class = EvaluationForm
+    template_name = "evaluation.html"
+    success_url = reverse_lazy("evaluation")
+
+    def get_context_data(self, **kwargs):
+        """Prepare the data for the evaluation page."""
+        context = super().get_context_data(**kwargs)
+        config = self.initiate_evaluation_configuration()
+        activity_key = config.get("activity_key")
+
+        # Query the database to get all traces
```

Comment:
Can we set up a function for this? Seems to be quite useful. Than writing that in get_context

Reply:
I implemented initiate_evaluation_configuration() to circumvent the need for an orchestrator instance, since evaluation should not need a running instance of it. the initiation and updating of the configuration in Filter- and ResultsView relies on the ExtractionConfiguration class from orchestrator.py and some functions from the Orchestrator class. This is why I made this function that utilizes session_data as a buffer instead. Do you think it might be needed elsewhere?

Reply:
No but we might. Just decide for urself.  Was just a sugestion. I just thought that it seems quiet useful to have a function instead of plaincoding in get context.